tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/iam/UserPermissionsSpec.scala

```diff
@@ -0,0 +1,28 @@
+package ch.epfl.bluebrain.nexus.tests.iam
+
+import akka.http.scaladsl.model.StatusCodes
+import ch.epfl.bluebrain.nexus.delta.kernel.utils.UrlUtils.encode
+import ch.epfl.bluebrain.nexus.tests.BaseSpec
+import ch.epfl.bluebrain.nexus.tests.Identity.userPermissions.{UserWithNoPermissions, UserWithPermissions}
+import ch.epfl.bluebrain.nexus.tests.iam.types.Permission.Resources
+
+class UserPermissionsSpec extends BaseSpec {
+
+  private def urlFor(permission: String, project: String) =
+    s"/user/permissions/$project?permission=${encode(permission)}"
+
+  "if a user does not have a permission, 403 should be returned" in {
+    deltaClient.head(urlFor("resources/read", "org/project"), UserWithNoPermissions) { response =>
```

Comment:
In the integration test, we try to generate randomly the org and project to avoid collisions between tests